162_find_peak_element.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+
+
+class Solution:
+    def findPeakElement(self, nums: List[int]) -> int:
```

Comment:
Your program is too complicated. Modify it with reference to solution or discuss.